gradle/dokka.gradle.kts

```diff
@@ -64,6 +66,17 @@ if (project.name == "kotlinx-coroutines-core") {
             val jvmMain by getting {
                 makeLinkMapping(project.file("jvm"))
             }
+
+            val isNewWasmTargetEnabled = isKotlinVersionAtLeast(kotlin_version, 1, 9, 20)
```

Comment:
Taking into account `1.9.20` is in Beta already, I suggest updating to Beta right in this branch and get rid of all the `if`s. We'll take care of updating it further and will release coroutines after `1.9.20` anyway

Reply:
WDYT?